tests/core/test_snapshot_evaluator.py

```diff
@@ -181,19 +181,11 @@ def x(evaluator, y=None) -> None:
         table_description=None,
     )
 
-    adapter_mock.create_table.assert_has_calls(
-        [
-            call(
-                snapshot.table_name(is_deployable=False),
-                column_descriptions=None,
-                **common_kwargs,
-            ),
-            call(
-                snapshot.table_name(),
-                column_descriptions={},
-                **common_kwargs,
-            ),
-        ]
+    # Create will be called once and only prod table will be created
```

Comment:
I've noticed we've modified all these tests, but have we added any which ensures that a dev table actually gets created?

Reply:
I added the `test_migrate_snapshot` test which validates a dev table been created for a forward only model but I will try to think of other use cases as well and add more

Reply:
Added unit tests for each case of creating dev preview tables, if you want to take another look @izeigerman

Reply:
Looks good, thanks 👍